packages/components/src/Workflow/subcomponents/Header/Header.tsx

```diff
@@ -1,6 +1,12 @@
 import React from "react"
-import { HeaderRoot, Branding, Titles, Actions } from "./components"
-import { WorkflowActionsProps, WorkflowTitlesProps } from "./"
+import {
+  HeaderRoot,
+  Branding,
+  Titles,
+  Actions,
+  WorkflowTitlesProps,
+  WorkflowActionsProps,
+} from "./components"
```

Comment:
We'll need to do this for the future release Workflow too 😉

Reply:
Odd, the Future one didn't complain